swift/Samples/AlertContainer/Sources/AlertContainerViewController.swift

```diff
@@ -0,0 +1,258 @@
+            addSubview(title)
+            
+            message.text = alert.message
+            addSubview(message)
+            
+            let buttonStackView = setupButtons(actions: alert.actions)
+            addSubview(buttonStackView)
+            
+            var constraints: [NSLayoutConstraint] = []
+            
+            constraints.append(title.topAnchor.constraint(equalTo: topAnchor, constant: 10))
+            constraints.append(title.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -10))
+            constraints.append(title.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 10))
+            constraints.append(title.heightAnchor.constraint(equalToConstant: 25))
+            
+            constraints.append(message.topAnchor.constraint(equalTo: title.bottomAnchor, constant: 10))
+            constraints.append(message.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -10))
+            constraints.append(message.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 10))
+            
+            constraints.append(buttonStackView.topAnchor.constraint(equalTo: message.bottomAnchor, constant: 15))
+            constraints.append(buttonStackView.bottomAnchor.constraint(equalTo: bottomAnchor, constant: 0))
+            constraints.append(buttonStackView.trailingAnchor.constraint(equalTo: trailingAnchor, constant: 0))
+            constraints.append(buttonStackView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 0))
+            constraints.append(buttonStackView.heightAnchor.constraint(greaterThanOrEqualToConstant: 50))
+            
+            addConstraints(constraints)
+        }
+    }
+    
+    private func setupButtons(actions: [AlertAction]) -> UIStackView {
+        let buttonStackView = UIStackView()
+        buttonStackView.axis = actions.count == 2 ? .horizontal : .vertical
+        buttonStackView.distribution = .fillEqually
+        buttonStackView.alignment = .fill
+        buttonStackView.translatesAutoresizingMaskIntoConstraints = false
+        
+        for action in actions {
+            let alertButton = AlertButton(action: action)
+            alertButton.backgroundColor = backgroundColor
+            alertButton.layer.borderColor = UIColor.gray.cgColor
+            alertButton.layer.borderWidth = 0.2
+            alertButton.translatesAutoresizingMaskIntoConstraints = false
+            
+            buttonStackView.addArrangedSubview(alertButton)
+        }
+        
+        return buttonStackView
+    }
+    
+    @available(*, unavailable)
+    public required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+private final class AlertButton: UIButton {
+    private var action: AlertAction
+    private var actionHandler: (() -> Void)?
+    
+    required init(action: AlertAction) {
+        self.action = action
+        super.init(frame: .zero)
+        commonInit()
```

Comment:
If this is the only place it's called from, maybe remove this and do the init directly?